### PR TITLE
fix: npx script

### PR DESCRIPTION
**Pull request by gyfchong**
Because I don't know how NPX works

### CODE REVIEW

File: packages/button/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kaizen/button",
-  "version": "2.1.0",
+  "version": "2.2.0",
```
Comment (HeartSquared) [inline packages/button/package.json:3]: Is this meant to be bumping?

File: packages/button/CHANGELOG.md [hunk 2]
```diff
@@ -1,834 +1,435 @@
 # Change Log
 
+## 2.2.0
```
Comment (gyfchong) [inline packages/button/CHANGELOG.md:3]: @HeartSquared how do I revert this file to `main` lol

Comment (HeartSquared) [reply packages/button/CHANGELOG.md:3]: https://stackoverflow.com/questions/37972753/git-reset-single-file-in-feature-branch-to-be-the-same-as-in-master

This? I wonder if you could reset hard on the single file 🤔 

Alt, you barely have any changes, so you could always just soft reset to main, then only commit the changes you want :)

Comment (gyfchong) [reply packages/button/CHANGELOG.md]: Yeah this worked:
```git checkout origin/master packages/button/CHANGELOG.md```

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**